bindings/pydeck/pydeck/frontend_semver.py

```diff
@@ -1 +1 @@
-DECKGL_SEMVER = "~8.5.*"
+DECKGL_SEMVER = '8.8.0-alpha.6'
```

Comment:
@ajduberstein can confirm, but I thought we usually had this file point to the current non-alpha release, which in this case would be
```suggestion
DECKGL_SEMVER = '8.7.*'
```

Reply:
If I do that, I see:
```
Unhandled Promise Rejection: Error: Module @deck.gl/jupyter-widget, semver range 8.7.* is not registered as a widget module
```

Reply:
Set this to `8.7.*` as suggested, but I believe anyone running from source would need to update this to be compatible with the jupyter-widget version (see #7025).

Reply:
Hmm. I think you're right that on the development branch, this has to match the development version of `@deck.gl/jupyter-widget`. So I think we need either:

- This should leave as-is or change to `8.8.*` if the star works with alpha versions.
- Cherry-pick these changes to the 8.7-release branch, but changing this to `8.7.*` there
- Publish the next version of pydeck from that branch, so that it uses the currently-released version of deck

Note that since `@deck.gl/jupyter-widgets` is a JS packaged published at the same time as every other JS package, this would require a patch release of the JS packages as well. cc @Pessimistress

Reply:
Right, forgot to mention that `~8.8.*` doesn't work either as alphas are strictly less than `8.8.0`. That was the reason I initially moved the work to the `8.7-release` branch.

Reply:
Makes sense. At least this way we have the code on the master branch for the future, and we can cherry pick for a release now.